third_party/toolchains/BUILD

```diff
@@ -56,6 +56,14 @@ platform(
             name: "dockerNetwork"
             value: "off"
         }
+        properties: {
```

Comment:
Looks like this is being set for all our RBE tests, not just ASAN?

Reply:
yes, Igor Foox mentioned that it should be applied to all RBE builds